src/mono/wasm/debugger/tests/debugger-test/debugger-evaluate-test.cs

```diff
@@ -496,6 +496,59 @@ public static void EvaluateLocals()
         }
     }
 
+    public static class EvaluateBrowsableProperties
```

Comment:
Not for this PR, but does this work with static fields, properties?

Reply:
Checked and for static fields it does work, for static properties - doesn't. I will create an issue on it.

Reply:
This case requires more effort than I thought. It requires fixing the way custom accessors' values are displayed, only then we can proceed to fixing Browsable. I am opening an issue #62774.